components/Ballot/OfficeRaces.tsx

```diff
@@ -0,0 +1,52 @@
+import { RaceResult } from "generated";
+import dynamic from "next/dynamic";
+import styles from "../Layout/Layout.module.scss";
+import Race from "./Race";
+
+const Scroller = dynamic(() => import("components/Scroller/Scroller"), {
+  ssr: false,
+});
```

Comment:
I tried using a named export from the scroller component but this next dynamic loader (which is akin to top level await) threw a compiler error.  I think we need to dynamically import it in whichever files we need it otherwise the next server will throw errors.  I left a comment in the other file where this pattern was used: 

> Note: this is a dynamic import because the react-horizontal-scrolling-menu uses useLayoutEffect which is not supported by the server.